src/main/java/org/mifos/connector/mpesa/dto/PaybillRequestDTO.java

```diff
@@ -0,0 +1,173 @@
+package org.mifos.connector.mpesa.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+//    {
+//        "TransactionType":"Pay Bill",
+//        "TransID":"RKTQDM7W6S",
+//        "TransTime":"20191122063845",
+//        "TransAmount":"10",
+//        "BusinessShortCode":"600638",
+//        "BillRefNumber":"A123",
+//        "InvoiceNumber":"",
+//        "OrgAccountBalance":"49197.00",
+//        "ThirdPartyTransID":"",
```

Comment:
remove comments

Reply:
Its a comment for request payload for reference.